apps/docs/src/app/core/component-docs/wizard/examples/wizard-mobile-example.component.ts

```diff
@@ -26,6 +26,16 @@ export class WizardMobileExampleComponent {
     example2step2status: WizardStepStatus = 'upcoming';
     example2step3status: WizardStepStatus = 'upcoming';
 
+    statusChanged(exampleNumber: number, stepNumber: number, event: any): void {
+        if (event === 'current') {
+            if (exampleNumber === 1) {
+                this.example1goToStep(stepNumber);
+            } else if (exampleNumber === 2) {
+                this.example2goToStep(stepNumber);
+            }
+        }
```

Comment:
please decrease nesting
```suggestion
        if (event !== 'current') {
            return;
        }

        if (exampleNumber === 1) {
            this.example1goToStep(stepNumber);
        } else if (exampleNumber === 2) {
            this.example2goToStep(stepNumber);
        }
```